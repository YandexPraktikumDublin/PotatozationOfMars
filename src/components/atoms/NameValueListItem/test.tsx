import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { NameValueListItem } from '.'

describe('<NameValueListItem />', () => {
  const name = 'Test name'
  const value = 'Test value'

  it('should renders correct <NameValueListItem />', () => {
    const wrapper = shallow(<NameValueListItem name={name} value={value} />)

    expect(wrapper.contains(name)).toBeTruthy()
    expect(wrapper.contains(value)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
