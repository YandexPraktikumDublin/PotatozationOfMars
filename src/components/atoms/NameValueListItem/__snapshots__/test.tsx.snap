// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NameValueListItem /> should renders correct <NameValueListItem /> 1`] = `
<li
  className="flex justify-between border-b border-primary mb-4 last:mb-0 dark:border-white"
>
  <span>
    Test name
  </span>
  <span>
    Test value
  </span>
</li>
`;
